packages/backend-defaults/src/entrypoints/database/DatabaseManager.ts

```diff
@@ -89,6 +89,27 @@ export class DatabaseManagerImpl {
     return { getClient, migrations: { skip } };
   }
 
+  /**
+   * Method to be called during shutdown to destroy all known connections.
+   */
+  async shutdown(deps: { logger: LoggerService }): Promise<void> {
+    const pluginIds = Array.from(this.databaseCache.keys());
+    await Promise.all(
+      pluginIds.map(async pluginId => {
+        const connection = await this.databaseCache.get(pluginId);
+        if (connection) {
```

Comment:
can it ever be empty?

Reply:
![9593yg](https://github.com/user-attachments/assets/03adee88-2e58-4021-b682-17ae4be07851)